muscle-tendon-complex/solver-opendihu/src/tendon-solver.cpp

```diff
@@ -0,0 +1,22 @@
+#include <Python.h>
+#include <cstdlib>
+#include <iostream>
+
+#include <iostream>
+#include "easylogging++.h"
+
+#include "opendihu.h"
```

Comment:
```suggestion
#include <iostream>
#include <easylogging++.h>
#include <opendihu.h>
```

- avoid duplicate headers
- use `< >` instead of `" "`, as you are not intending to search local directories first (otherwise, these files would have been in the same directory).